### PR TITLE
Added apiNG

### PR DESCRIPTION
**Pull request by JohnnyTheTank**
Added https://github.com/JohnnyTheTank/apiNG

### CODE REVIEW

File: files/aping/update.json [hunk 1]
```diff
@@ -0,0 +1,8 @@
+{
+  "packageManager": "github",
+  "name": "apiNG",
+  "repo": "JohnnyTheTank/apiNG",
+  "files": {
+    "include": ["dist/*"]
```
Comment (jimaek) [inline files/aping/update.json]: Change it to `dist/*.min.*` please

Comment (JohnnyTheTank) [inline files/aping/update.json]: done

**Comment by jimaek** [comment]
@jsdelivrbot squash and merge

**Comment by jimaek** [comment]
@jsdelivrbot squash and merge

**Comment by jimaek** [comment]
@jsdelivrbot squash and merge

**Comment by megawac** [comment]
What did you do to my bot @jimaek?!

**Comment by jimaek** [comment]
It cant merge this PR for some reason :( Not sure if others have the same issue.
Havent logged in to the bot server for months

**Comment by jimaek** [comment]
@jsdelivrbot squash and merge

**Comment by JohnnyTheTank** [comment]
I close this PR and will create a new one

**Pull request closed without merging**